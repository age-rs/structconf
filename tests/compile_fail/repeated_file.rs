//! Shouldn't compile because an identifier is repeated

use structconf::StructConf;

#[derive(StructConf)]
struct Config {
    #[conf(no_short, no_long, file = "abc")]
    pub name1: bool,
    #[conf(no_short, no_long, file = "abc")]
    pub name2: bool,
}

fn main() {}
