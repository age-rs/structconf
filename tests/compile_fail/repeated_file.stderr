error: The `short` attribute with value `n` is repeated with another field. Try using the `no_short` and `short` attributes.
  --> $DIR/repeated_file.rs:10:9
   |
10 |     pub name2: bool,
   |         ^^^^^
